finance/bitcoin/Portfile

```diff
@@ -21,8 +21,18 @@ master_sites            ${homepage}bin/bitcoin-core-${version}/
 checksums           rmd160  a11078445daad75bbe2032da307d292670eece55 \
                     sha256  34de2dbe058c1f8b6464494468ebe2ff0422614203d292da1c6458d6f87342b4
 
-depends_build	port:autoconf port:automake port:libtool port:pkgconfig port:protobuf-cpp port:python36
-depends_lib	port:boost port:libevent port:miniupnpc port:openssl port:zmq
+depends_build	port:autoconf		\
+		port:automake		\
+		port:libtool		\
+		port:pkgconfig		\
+		port:protobuf-cpp	\
+		port:python36
+
+depends_lib	port:boost		\
+		port:libevent		\
+		port:miniupnpc		\
+		port:openssl		\
+		port:zmq
```

Comment:
Prior to your changes, the second column of the dependency lines lined up with the second column of the rest of the portfile; now it doesn't.